.github/workflows/build_and_test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        os: [ ubuntu-20.04, windows-latest, macos-latest ]
```

Comment:
why downgrade is required?

Reply:
This is because GitHub switched to new version of ubuntu, which is not supported by current version librdkafka. Actually this is not downgrading but preserving the same version.